packages/video_player/README.md

```diff
@@ -0,0 +1,55 @@
+# video_player_tizen
+
+The Tizen implementation of [`video_player`](https://github.com/flutter/plugins/tree/master/packages/video_player).
+
+## Required privileges
+
+To use this plugin in a Tizen application, the mediastorage, externalstorage and internet privileges are required. Add below lines under the `<manifest>` section in your `tizen-manifest.xml` file.
+
+```xml
+<privileges>
+  <privilege>http://tizen.org/privilege/mediastorage</privilege>
+  <privilege>http://tizen.org/privilege/externalstorage</privilege>
+  <privilege>http://tizen.org/privilege/internet</privilege>
+</privileges>
+```
+
+- The mediastorage privilege (`http://tizen.org/privilege/mediastorage`) must be added to play video files located in the internal storage.
+- The externalstorage privilege (`http://tizen.org/privilege/externalstorage`) must be added to play video files located in the external storage.
+- The internet privilege (`http://tizen.org/privilege/internet`) must be added to play any URLs from network.
+
+For details, see [Security and API Privileges](https://docs.tizen.org/application/dotnet/tutorials/sec-privileges).
+
+## Usage
+
+This package is not an _endorsed_ implementation of `video_player`. Therefore, you have to include `video_player_tizen` alongside `video_player` as dependencies in your `pubspec.yaml` file.
+
+```yaml
+dependencies:
+  video_player: ^1.0.1
+  video_player_tizen: ^1.0.0
+```
+
+Then you can import `video_player` in your Dart code:
+
+```dart
+import 'package:video_player/video_player.dart';
+```
+
+For how to use the plugin, see https://github.com/flutter/plugins/tree/master/packages/video_player/video_player#usage.
+
+## Supported devices
+
+This plugin is supported on these types of devices:
+
+- Galaxy Watch (running Tizen 4.0 or later)
```

Comment:
@xiaowei-guan Is this plugin supported on Tizen 4.0 now? Haven't you told me it only supports Tizen 5.5 and Tizen 6.0? If so, we don't need this section at all because the limitation is already covered by https://github.com/flutter-tizen/flutter-tizen#supported-devices.

Reply:
I can build the video player on tizen 4.0 successfully. I think it support tizen 4.0
I test the example on watch with tizen 4.0, but it failed to launch the app.
It seems flutter app can't be launched on both real device and emulator.

I got some error log:
ConsoleMessage( 4291): FlutterApplication.cs: Run(39) > Unhandled exception: System.IO.FileLoadException: Could not load file or assembly 'System.Runtime, Version=4.2.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)
E/ConsoleMessage( 4291): File name: 'System.Runtime, Version=4.2.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
E/ConsoleMessage( 4291):    at Runner.App.OnCreate()
E/ConsoleMessage( 4291):    at Tizen.Applications.CoreBackend.UICoreBackend.OnCreateNative(IntPtr data)
E/ConsoleMessage( 4291):    at Tizen.Applications.CoreBackend.UICoreBackend.Run(String[] args)
E/ConsoleMessage( 4291):    at Tizen.Applications.CoreApplication.Run(String[] args)
E/ConsoleMessage( 4291):    at Tizen.Applications.CoreUIApplication.Run(String[] args)
E/ConsoleMessage( 4291):    at Tizen.Flutter.Embedding.FlutterApplication.Run(String[] args)
E/ConsoleMessage( 4291):    at Runner.App.Main(String[] args)
E/ConsoleMessage( 4291):

Reply:
That error has nothing to do with Flutter. Did you also set the value of .NET API version ("api-version" attribute of "ui-application" element) to "4"?

```xml
<ui-application appid="org.tizen.video_player_tizen_example" exec="Runner.dll" type="dotnet"
  multiple="false" taskmanage="true" nodisplay="false" api-version="6" launch_mode="single">
                                                                    ^ here
```

Reply:
yes, I changed both "api-version" attribute of "ui-application" element

Reply:
How about the Runner.csproj file? The target framework moniker should be "tizen40".

Reply:
I change target framework to "tizen40", I can see the log of flutter, but there is crash.

E/ConsoleMessage( 7408): ../../../../flutter/shell/platform/tizen/flutter_tizen.cc: FlutterDesktopRegistrarGetTextureRegistrar(221) > Tizen DesktopTexture support is not implemented yet.
E/ConsoleMessage( 7408): ../../../../flutter/shell/platform/tizen/flutter_tizen.cc: FlutterDesktopRegistrarGetTextureRegistrar(221) > Tizen DesktopTexture support is not implemented yet.
E/ConsoleMessage( 7408): ../../../../flutter/shell/platform/tizen/flutter_tizen.cc: FlutterDesktopRegistrarGetTextureRegistrar(221) > Tizen DesktopTexture support is not implemented yet.
E/ConsoleMessage( 7408):
E/ConsoleMessage( 7408): (process:7408): GLib-CRITICAL **: g_hash_table_foreach_remove: assertion 'hash_table != NULL' failed
E/ConsoleMessage( 7408):
E/ConsoleMessage( 7408): (process:7408): GLib-CRITICAL **: g_hash_table_foreach_remove: assertion 'hash_table != NULL' failed
......
E/ConsoleMessage( 7408): (process:7408): GLib-CRITICAL **: g_hash_table_foreach_remove: assertion 'hash_table != NULL' failed
E/ConsoleMessage( 7408): crashed [1514812816] processname=Runner.dll, pid=7408, tid=7668, signal=6
E/ConsoleMessage( 7408): [sys-assert] callstack generated by crash-stack

Reply:
It seems there was a problem during Flutter 2.0 migration. Any app that previously could be launched Tizen 4.0 device doesn't any more run on the same device with this (https://github.com/flutter-tizen/engine/releases/tag/113f51b) version of engine.